src/Classes/Factories/AddHiringPartnerPageControllerFactory.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Portal\Factories;
+
+use Portal\Controllers\AddHiringPartnerPageController;
+
+class AddHiringPartnerPageControllerFactory
+{
+    /**
+     * @param $c
+     * @return AddHiringPartnerPageController
+     */
+    public function __invoke($c)
```

Comment:
What's $c? Is it a bird? Is it a plane? The factories the previous cohort made had typehinting on them so we should continue the example

Reply:
the previous cohort also named it $container